app/src/renderer/components/wallet/PageTransactions.vue

```diff
@@ -56,8 +56,8 @@ export default {
     time
   }),
   computed: {
-    ...mapState([`transactions`]),
     ...mapGetters([
+      `transactions`,
```

Comment:
where are we using this ?

Reply:
at the header we are forwarding `loading`, `loaded` and `error` to the Page component, I found a getter less invasive than the mapState, and this was the only place in the whole codebase where mapState was used so, I proposed to drop it :)